src/raft_log.rs

```diff
@@ -1321,19 +1321,19 @@ mod test {
                 1000,
                 vec![300, 500, 800, 900],
                 vec![700, 500, 200, 100],
-                true,
+                false,
```

Comment:
Why this change?

Reply:
Because I think this case shouldn't panic.